prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -328,29 +332,49 @@ class FHIRRouter(
             // JURIS FILTER
             //  default: allowNone
             var passes = evaluateFilterCondition(getJurisFilters(receiver, orgFilters), bundle, false)
+
             // QUALITY FILTER
-            //  default: must have message id, patient last name, patient first name, dob, specimen type
-            //           must have at least one of patient street, zip code, phone number, email
-            //           must have at least one of order test date, specimen collection date/time, test result date
-            passes = passes &&
-                evaluateFilterCondition(
-                    getQualityFilters(receiver, orgFilters),
+            if (passes) {
```

Comment:
Rename passes to something more meaningful since it is now outside of the passes &&.  notFiltered?